tarrif/src/main/java/pl/edu/agh/tarrif/tarrifContract/Tarrif.java

```diff
@@ -0,0 +1,134 @@
5280156100cb5781602001602082028036833780820191505090505b509050600060018054905067ffffffffffffffff8111156100ef576100ee610503565b5b60405190808252806020026020018201604052801561011d5781602001602082028036833780820191505090505b509050600060018054905067ffffffffffffffff81111561014157610140610503565b5b60405190808252806020026020018201604052801561016f5781602001602082028036833780820191505090505b50905060005b600180549050811015610264576001818154811061019657610195610532565b5b9060005260206000209060030201600001548482815181106101bb576101ba610532565b5b602002602001018181525050600181815481106101db576101da610532565b5b906000526020600020906003020160010154838281518110610200576101ff610532565b5b602002602001018181525050600181815481106102205761021f610532565b5b90600052602060002090600302016002015482828151811061024557610244610532565b5b602002602001018181525050808061025c90610590565b915050610175565b50828282955095509550505050909192565b61027e6102ee565b6000604051806060016040528084815260200183815260200142815250905060018190806001815401808255809150506001900390600052602060002090600302016000909190919091506000820151816000015560208201518160010155604082015181600201555050505050565b60008054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff161461037c576040517f08c379a000000000000000000000000000000000000000000000000000000000815260040161037390610635565b60405180910390fd5b565b600081519050919050565b600082825260208201905092915050565b6000819050602082019050919050565b6000819050919050565b6103bd816103aa565b82525050565b60006103cf83836103b4565b60208301905092915050565b6000602082019050919050565b60006103f38261037e565b6103fd8185610389565b93506104088361039a565b8060005b8381101561043957815161042088826103c3565b975061042b836103db565b92505060018101905061040c565b5085935050505092915050565b6000606082019050818103600083015261046081866103e8565b9050818103602083015261047481856103e8565b9050818103604083015261048881846103e8565b9050949350505050565b600080fd5b6104a0816103aa565b81146104ab57600080fd5b50565b6000813590506104bd81610497565b92915050565b600080604083850312156104da576104d9610492565b5b60006104e8858286016104ae565b92505060206104f9858286016104ae565b9150509250929050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b600061059b826103aa565b91507fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff82036105cd576105cc610561565b5b600182019050919050565b600082825260208201905092915050565b7f41636365732064656e696564202d206e6f2061646d696e206163636573730000600082015250565b600061061f601e836105d8565b915061062a826105e9565b602082019050919050565b6000602082019050818103600083015261064e81610612565b905091905056fea26469706673582212208ed516d543c0d66529540893814aeb5c132688baa2f96352bfc4a4195b2a1b0164736f6c634300080e0033";
```

Comment:
I'm not completely against that infinitely long line, but If possible try to change it to finate long line.